src/churchtoolsClient.js

```diff
@@ -33,7 +34,11 @@ class ChurchToolsClient {
             return response;
         });
 
+        this.setRateLimitInterceptor();
```

Comment:
Wenn ich es richtig sehe, haben wir dann keine Möglichkeit mehr zu sagen: Ich will für meine Anwendung keinen RateLimitInterceptor haben.

In den meisten Fällen erwarten wir ja kein Rate Limit. Bei allen Requests in der App zum Beispiel. Wenn hier ein 429er auftreten sollte, ist etwas richtig kaputt, und ich denke wir wollen lieber einen Fehler anzeigen, als dass 30 Sekunden nichts passiert.

Man könnte diese Methode also z.B. einfach nicht im Konstruktor aufrufen, sondern bei Bedarf von der Anwendung, die den ChurchToolsClient instantiiert und benutzt. Oder umgekehrt, im Konstruktor wird es gesetzt, aber wir haben über eine Methode die Möglichkeit, den Interceptor wieder zu ejecten.

Reply:
erledigt. Ist jetzt per Default aus, kann insertet werden via `setRateLimitInterceptor`